ofApp.cpp

```diff
@@ -1,387 +0,0 @@
-    rightObj.population = json[rightObj.random]["population"].asInt();
-    rightObj.x = 0;
-
-    // the following code we only need one copy of - therefore I attached it to the leftObject but this is not relevant.
-    leftObj.areaOrPopulation = 'I';
-    leftObj.updateBoolean = true;
-    leftObj.score = 0;
-    leftObj.highscore = 0; // we acc only need 1 of these 
-    leftObj.landlocked = json[leftObj.random]["landlocked"].asString(); // for some reason it did not want to pull in a bool - kept giving me a 0
-    leftObj.landlockedLife = '1'; // have one life
-
-    // sets the starting screen to home
-    screen = screens::home;
-
-    backgroundImage.load("data\\red.png"); // I already had these images from when I was planning and picked the colours so I used them as images instead of drawing a rectangle of colour x - hasnt made a difference so its ok
-    gameOverTextAreaImage.load("data\\yellow.png");
-    leftOptionImage.load("data\\blue.png");
-    rightOptionImage.load("data\\orange.png");
-
-    leftOptionButton.set(0,0,0,0); // dont need them drawn yet since they are not for the home screen so set their information to 0
-    rightOptionButton.set(0, 0, 0, 0);
-    gameOverTextAreaButton.set(0,0,0,0);
-    choice1.set(0, 0, 0, 0);
-    choice2.set(0, 0, 0, 0);
-
-    font.load("data\\font.ttf", 28); // this font I used for titles and important info
-    titleText = "Welcome to the Higher or Lower Game!";
-
-    font2.load("data\\font.ttf", 16); // this font I used for less important stuff like the description
-
-    descriptionText = "Below are two buttons for slightly different experiences. One of them, the one on the left will start the game with population as the variable. So you need to pick which country you think has the larger population. The other button, the one on the right, uses the area. So you need to pick which country has a larger surface area.";
-    wrappedTextTeaCup = wrapString(descriptionText, 1100, false); // this calls wrapString and puts in the description, a width id like to cap it at, and a boolean to decide whether or not its a header. true for header false for not header.
-
-    populationButton.load("data\\blue_population.png");
-    areaButton.load("data\\yellow_area.png");
-    metalGearSolid.load("data\\mgs.jpg");
-
-    leftObj.colour = ""; // .colour is used to provide user feedback so they know they are selecting one or the other button.
-    rightObj.colour = "";
-
-}
-
-void ofApp::draw(){ // anything which needs to be represented in the GUI should be in this draw area.
-    // set color to white this ensures no tint is placed on image which is drawn next
-    ofSetColor(255);
-    // this if checks to see what current screen the user is on. there will be a lot of these since they are essential conditions for which code can be applied on. 
-    if (screen == screens::home) {
-        backgroundImage.draw(0, 0, 1200, 750); // this should cover the whole home screen at all times
```

Comment:
Again could refactor these screens to be callable functions that draw each page, this could clean up the draw function and make it more readable